src/lib/route/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './utils';
+export * from './data';
```

Comment:
Wir müssen schauen, ob wir hier wirklich alles exportieren wollen. Ich denke es wäre sinnvoll hier nur punktuell zu exportieren, da diese datei dann im gesamten in der public-api landet.